core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -83,6 +83,14 @@
   RPC_SASL_QOP("rpc.sasl.qop", "auth", PropertyType.STRING,
       "The quality of protection to be used with SASL. Valid values are 'auth', 'auth-int',"
           + " and 'auth-conf'"),
+  /**
+   * @since 2.1.0
+   */
+  SYSTEM_TOKEN_HASH_TYPE("system.token.hash.type", Constants.PW_HASH_ALGORITHM_OUTDATED,
```

Comment:
See my other comment about not making this configurable. However, since it is configurable, it must be the same across all servers, or they won't be able to talk to each other. In general, that would mean making this an `instance.*` property. However, since the way we enforce these is through the same-ness of the system credentials, if this is different across the cluster, then they won't be able to talk to each other anyway. The other property prefix we use for system-wide properties that can be different are `general.*`.

Reply:
I personally think `instance` seems an appropriate prefix, so it would be `instance.system.token.hash.type`.

Reply:
Thank you for the suggestions. Adding the `instance.*` prefix.